pom.xml

```diff
@@ -43,11 +43,17 @@
     </licenses>
 
     <properties>
+        <argLine></argLine>
```

Comment:
jacoco `prepare-agent` needs to set this variable. if we don't have a default set then maven gets angry.